atom/common/electron_version.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2013 GitHub, Inc.
```

Comment:
Git thinks this is a new file, but it's just renamed and changed a lot

Reply:
can we please generate this file during build?